pages/home.py

```diff
@@ -780,6 +916,23 @@ def render_overview_total_figure(data, country):
 
     return functional_figure(func_df), functional_narrative(func_df)
 
+@callback(
+    Output("economic-breakdown", "figure"),
+    Output("economic-narrative", "children"),
+    Input("stored-data-func-econ", "data"),
+    Input("country-select", "value"),
+)
+def render_overview_econ_figure(data, country):
```

Comment:
should we put this functions above  the function that calls economic_figure and economic_narrative  ?